configuration/package_scripts/_generated/c8y-remote-access-plugin/apk/postinst

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 set -e
 
+
+
+
```

Comment:
Note, these blank sections are a result of a non-ideal implementation in the "generate.py" script which does not skip empty evaluated template blocks. This behaviour is not different to before, and other `postinst` scripts (mostly for apk packages) also have the same "empty scripts". This should be fixed, but it is not part of this PR.